pylua/test/test_timer.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import timeit
+import threading
+import pylua.timer as timer
+
+
+class TestPyTimer:
+
+    def _timer_task(self):
+        self._task_is_run = True
+        self._event.set()
+
+    def __init__(self):
+        self._interval = 50  # interval in milliseconds
+        self._timer = timer.PyTimer(self._timer_task, self._interval)
+        self._task_is_run = False
+        self._event = threading.Event()
+
+    def _run_event(self, wait_time):
+        wait_time = timer.convert_milliseconds_to_seconds(wait_time)
+        self._event.clear()
+        self._event.wait(wait_time)
+
+    def _run_timer(self, interval=None):
+        self._task_is_run = False
+        started = timeit.default_timer()
+        if (interval is None):
+            wait_time = self._interval + 500
+            self._timer.start()
+        else:
+            wait_time = interval + 500
+            self._interval = interval
+            self._timer.start(interval)
+        self._run_event(wait_time)
+        assert self._task_is_run is True, "Task wasn't run"
+        delay = (timeit.default_timer() - started -
+                 timer.convert_milliseconds_to_seconds(self._interval))
+        assert delay < 0.06, "Timer doesn't work correctly"
+
+    def test_start_timer_with_parameter(self):
+        self._run_timer(50)  # interval in milliseconds
+
+    def test_start_timer_without_parameter(self):
+        self._run_timer()
+
+    def test_start_timer_periodicaly_with_parameter(self):
+        for i in range(0, 5):
+            self._run_timer(50)  # interval in milliseconds
+
+    def test_start_timer_periodicaly_without_parameter(self):
+        for i in range(0, 5):
+            self._run_timer()
+
+    def test_timer_set_interval(self):
```

Comment:
@dtrunov, am I right that you're testing python property feature? Why do think this is required?